packages/web-app-files/src/composables/resourcesViewDefaults/useResourcesViewDefaults.ts

```diff
@@ -65,7 +64,7 @@ export const useResourcesViewDefaults = <T, TT, TU extends any[]>(
   })
 
   const resourcesStore = useResourcesStore()
-  const { activeResources: storeItems } = storeToRefs(resourcesStore)
+  const storeItems = computed(() => resourcesStore.activeResources) as unknown as Ref<T[]>
```

Comment:
if we need this more often, it might make sense to add this a function to the resourcesStore... fine for now